### DISCUSSION TITLE
Mainnet Beta Release 1.4.0beta3

### DISCUSSION THREAD

**Discussion by vanarasiantonyswamy** [MinaProtocol/mina#13393] [Announcements]
The O(1) Labs team is proud to announce that 1.4.0beta3 is ready to test on Mainnet! This release includes a set of minor bugfixes, RFCs and new documentation, major testing and CI improvements, and code cleanup. The change has significant performance improvement on ledger export on user experience but include months of work to improve testing, processes, and code quality.

If you run into any issues, please include `Release: 1.4.0beta3` in the name of any issues you create. Please report any issues you encounter [here](https://github.com/MinaProtocol/mina/issues/new/choose) or join the Discussion below!

## Improvements/Features:
-  Faster ledger export #13316

### Complete Changelog since 1.3.1.2:
https://github.com/MinaProtocol/mina/compare/1.3.1.2...1.4.0beta3

### Debian Packages:
**Debian Repository:**
First, set up and update the `beta` Debian Repository for your platform with the following commands:
```
sudo rm /etc/apt/sources.list.d/mina*.list
echo "deb [trusted=yes] http://packages.o1test.net $(lsb_release -cs) beta" | sudo tee /etc/apt/sources.list.d/mina-beta.list
sudo apt-get update
```
Then, install the package(s) that you need:
**Mina Daemon:** `sudo apt-get install -y mina-mainnet=1.4.0beta3-1a8adfa`
**Mina Archive:** `sudo apt-get install -y mina-archive=1.4.0beta3-1a8adfa`
**New Mina Archive Schema:** [create_schema.sql](https://github.com/MinaProtocol/mina/blob/1a8adfa0d00f88c955e503f5f6eaef775f30e2be/src/app/archive/create_schema.sql)

### Docker Images:
Every docker image is built for each of the supported platforms, replace `CODENAME` in the images below with your choice of `bionic`, `focal`,  `buster`, or `bullseye` to select the base docker image.

**Mina Daemon:** `minaprotocol/mina-daemon:1.4.0beta3-1a8adfa-CODENAME-mainnet`
**Mina Archive:** `minaprotocol/mina-archive:1.4.0beta3-1a8adfa-CODENAME`
**New Mina Archive Schema:** [create_schema.sql](https://github.com/MinaProtocol/mina/blob/1a8adfa0d00f88c955e503f5f6eaef775f30e2be/src/app/archive/create_schema.sql)
**Mina Rosetta:** `minaprotocol/mina-rosetta:1.4.0beta3-1a8adfa-CODENAME`

### Sandbox Node: For testing in an isolated, single-node network without snarks
As of 1.2.0, the demo functionality is built into the standard daemon container, via the environment variable `RUN_DEMO=true`.
Just `docker run -it --name mina-demo -e RUN_DEMO=true minaprotocol/mina-daemon:1.4.0beta3-1a8adfa-CODENAME-devnet` and go!

### Step by Step Guide:
Check out [our documentation](https://docs.minaprotocol.com/node-operators/connecting-to-the-network) for complete instructions on using this version to connect to Mainnet.
*Docker only* Make sure to run with `--peer-list-url https://storage.googleapis.com/mina-seed-lists/mainnet_seeds.txt` or add `PEER_LIST_URL=https://storage.googleapis.com/mina-seed-lists/mainnet_seeds.txt` to your `.mina-env`.

If you are running the correct version on the correct network **MAINNET**, `mina client status` will show:
```
Chain id:     5f704cc0c82e0ed70e873f0893d7e06f148524e3f0bdae2afb02e7819a0c24d1
Git SHA-1:    1a8adfa0d00f88c955e503f5f6eaef775f30e2be
```
### Retiring
We are retiring support on `stretch`.
<hr /><em>This discussion was created from the release <a href='https://github.com/MinaProtocol/mina/releases/tag/1.4.0beta3'>Mainnet Beta Release 1.4.0beta3</a>.</em>

**Comment by Villiking** [comment]
good

**Comment by Mtest3** [comment]
hey
